spec/type/base/file_spec.rb

```diff
@@ -338,10 +338,25 @@
 end
 
 describe file('/etc/passwd') do
+  let(:stdout) { "644\r\n" }
```

Comment:
I tried to avoid using `let(:stdout)` but couldn't seem to get it working otherwise. I'm still not sure how the `be_mode` and `be_owned_by` matchers are working =/